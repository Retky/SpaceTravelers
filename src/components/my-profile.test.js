import { Provider } from 'react-redux';
import renderer from 'react-test-renderer'
import store from '../redux/configureStore';
import Profile from './my-profile';

describe(' My Profile Component Rendering ', () =>{
    it('renders correctly', () => {
      const tree = renderer.create(
        <Provider store={store}>
          <Profile />
        </Provider>,
      ).toJSON();
      expect(tree).toMatchSnapshot();
    });
})