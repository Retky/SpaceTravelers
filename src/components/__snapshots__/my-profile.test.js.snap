// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Missions Component Rendering  renders correctly 1`] = `
<section
  className="profile"
>
  <div
    className="profMissions"
  >
    <h2>
      My Missions
    </h2>
    <ul
      className="my"
    />
  </div>
  <div
    className="profRockets"
  >
    <h2>
      My Rockets
    </h2>
    <ul
      className="my"
    />
  </div>
</section>
`;

exports[` My Profile Component Rendering  renders correctly 1`] = `
<section
  className="profile"
>
  <div
    className="profMissions"
  >
    <h2>
      My Missions
    </h2>
    <ul
      className="my"
    />
  </div>
  <div
    className="profRockets"
  >
    <h2>
      My Rockets
    </h2>
    <ul
      className="my"
    />
  </div>
</section>
`;
